test/utils.js

```diff
@@ -0,0 +1,14 @@
+export const request = url => (
+    new Promise(resolve => {
+        const xhr = new XMLHttpRequest();
+        xhr.onreadystatechange = () => (
+            xhr.readyState === 4 && resolve(xhr)
+        );
+        xhr.open('GET', url, true);
+        xhr.send();
+    })
+);
+
+export default {
```

Comment:
Why here there are both export const and export default?

Reply:
so that you can import like so:
```js
import { request } from './utils';
...request()

// or

import utils from './utils';
utils.request()
```

without having to
```js
import * as utils from './utils'
```

I think you guys are using this pattern in HomeWeb in places